<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMachineWindow</class>
 <widget class="QWidget" name="AddMachineWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>434</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>434</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>710</width>
    <height>434</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active/>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Inter</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Добавление станка</string>
  </property>
  <widget class="QLabel" name="add_machine_label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>321</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить станок</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_line_edit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="type_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тип станка</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="brand_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Бренд</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="go_area_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Личный кабинет</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QComboBox" name="type_combo_box">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Тип станка</string>
   </property>
  </widget>
  <widget class="QComboBox" name="brand_combo_box">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Бренд</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
